# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/FragmentOrElement.h
# Commit: 95679f6ce254
# Full Hash: 95679f6ce2544b21ec9cfd0d19961058d60155d6
# Author: Frederic Wang <fred.wang@free.fr>
# Date: 2023-11-27 17:00:06
# Regressor Bug: 1807253
# File Overlap Count: 1
# Description:
#   Bug 1807253 - unreliable timing for content-visibility:auto, r=emilio
#   
#   Rewrite implementation of content-visibility: auto as defined in
#   https://github.com/w3c/csswg-drafts/issues/8542
#   
# ==============================================================================

diff -r 9ea04a2cc829 -r 95679f6ce254 dom/base/FragmentOrElement.h
--- a/dom/base/FragmentOrElement.h	Mon Nov 27 08:00:41 2023 +0000
+++ b/dom/base/FragmentOrElement.h	Mon Nov 27 08:26:46 2023 +0000
@@ -238,12 +238,16 @@
     /**
      * Whether the content of this element is relevant for the purposes
      * of `content-visibility: auto.
+     * Reflects 'relevant to the user' concept, see
+     * https://drafts.csswg.org/css-contain/#relevant-to-the-user.
      */
     Maybe<ContentRelevancy> mContentRelevancy;
 
     /**
      * Whether the content of this element is considered visible for
      * the purposes of `content-visibility: auto.
+     * Reflects 'proximity to the viewport' concept, see
+     * https://drafts.csswg.org/css-contain/#proximity-to-the-viewport.
      */
     Maybe<bool> mVisibleForContentVisibility;
 